runtime/drivers/duckdb/connectors.go

```diff
@@ -43,6 +43,20 @@ func (c *connection) Ingest(ctx context.Context, env *connectors.Env, source *co
 	return summary, err
 }
 
+func (c *connection) EstimateSize() (int64, bool) {
+	var paths []string
+	path := c.config.DSN
```

Comment:
The DSN may have options, like `xxx?opt1=val1`, so this will not work reliably. Need to use `c.config.DBFilePath`